### PR TITLE
Support more languages in Generate Code

### LINKED ISSUE #2990: Support more languages in Generate Code
### I have checked the following:

- [X] I've searched existing issues and found nothing related to my issue.

### Describe the feature you want to add

Since I usually work on Java projects, I want to generate Java code from the collections.
Then I checked `httpsnippet`, the library Bruno is using to generate code snippets, and saw that it supports more languages than the current ones.
Perhaps there should be a way to show all languages from `httpsnippet` in Bruno so that I can generate code snippets in any language `httpsnippet` supports.

### Mockups or Images of the feature

Screenshot_20240902_110210

### PR DESCRIPTION
**Pull request by HSGamer**
# Description

This PR adds a way to convert all targets from `httpsnippet` to languages in Bruno so that users can choose any language `httpsnippet` supports. That includes C, C#, Java, Kotlin, etc.

Screenshot_20240902_110210

### Contribution Checklist:

- [x] **The pull request only addresses one issue or adds one feature.**
- [x] **The pull request does not introduce any breaking changes**
- [x] **I have added screenshots or gifs to help explain the change if applicable.**
- [x] **I have read the [contribution guidelines](https://github.com/usebruno/bruno/blob/main/contributing.md).**
- [x] **Create an issue and link to the pull request.**

Close #2990

### CODE REVIEW

File: packages/bruno-app/src/utils/codegenerator/targets.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import { targets } from 'httpsnippet';
+
+export const getLanguages = () => {
+  const allLanguages = [];
+  console.log(Object.values(targets));
```
Comment (nikischin) [inline packages/bruno-app/src/utils/codegenerator/targets.js]: Console.log should be removed :)

Comment (HSGamer) [reply packages/bruno-app/src/utils/codegenerator/targets.js]: Done XD

**Comment by stefanstranger** [comment]
In which version of Bruno can I find this? I want to check if PowerShell is now supported.

**Comment by Its-treason** [comment]
> In which version of Bruno can I find this? I want to check if PowerShell is now supported.

It's not yet released. The next version will most likely be published tomorrow.

**Comment by HSGamer** [comment]
> In which version of Bruno can I find this? I want to check if PowerShell is now supported.

It's indeed supported

image

**Pull request merged**